### PR TITLE
Linted README

### PR DESCRIPTION
**Pull request by kopiro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,69 +2,67 @@
 
 > Use Ansible command line tools inside Docker in CI or Local Environment.
 
-
 ## Supported tags and respective `Dockerfile` links
 
-  * [`latest`](https://github.com/gadiener/ansible/blob/master/Dockerfile) [![](https://images.microbadger.com/badges/image/gdiener/ansible:latest.svg)](http://microbadger.com/images/gdiener/ansible:latest)
-  * [`1`, `1.0`, `1.0.0`](https://github.com/gadiener/ansible/blob/1.0.0/Dockerfile) [![](https://images.microbadger.com/badges/image/gdiener/ansible:1.svg)](http://microbadger.com/images/gdiener/ansible:1)
-
+* [`latest`](https://github.com/gadiener/ansible/blob/master/Dockerfile) [![](https://images.microbadger.com/badges/image/gdiener/ansible:latest.svg)](http://microbadger.com/images/gdiener/ansible:latest)
```
Comment (kopiro) [inline README.md]: Removed initial space.

File: README.md [hunk 2]
```diff
@@ -2,69 +2,67 @@
...
+* [`1`, `1.0`, `1.0.0`](https://github.com/gadiener/ansible/blob/1.0.0/Dockerfile) [![](https://images.microbadger.com/badges/image/gdiener/ansible:1.svg)](http://microbadger.com/images/gdiener/ansible:1)
 
 ## How to use
 
 ### Docker run
 
- * [ansible](https://docs.ansible.com/ansible/2.5/cli/ansible.html): Define and run a single task ‘playbook’ against a set of hosts.
```
Comment (kopiro) [inline README.md]: Removed initial space.

File: README.md [hunk 3]
```diff
@@ -2,69 +2,67 @@
...
+* [ansible](https://docs.ansible.com/ansible/2.5/cli/ansible.html): Define and run a single task ‘playbook’ against a set of hosts.
 
 ```bash
-$ docker run -v $(pwd):/playbook/ -it gdiener/ansible ansible all -m ping
```
Comment (kopiro) [inline README.md]: Do not put dollar signs in commands.

However, an exception is made when there is a need to distinguish between typed commands and command output.

Read more here:

https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md014---dollar-signs-used-before-commands-without-showing-output

http://www.cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code

File: README.md [hunk 4]
```diff
@@ -96,20 +94,19 @@ Options:
...
-$ rm /usr/local/bin/ansible-aliases
+ansible-aliases --remove
+rm /usr/local/bin/ansible-aliases
 ```
 
-
 ## Configuration
 
-### Enviroment variables
+### Environment variables
```
Comment (kopiro) [inline README.md]: Typo.

File: README.md [hunk 5]
```diff
@@ -129,7 +126,6 @@ You can bind a volume to `/etc/ansible` to share your local conficuration with A
 
 @todo: I'm waiting for your pull request 😉
 
-
```
Comment (kopiro) [inline README.md]: Removing unnecessary line.

**Comment by gadiener** [comment]
Thanks

**Pull request merged**